modules/ROOT/partials/beta-disclaimer.adoc

```diff
@@ -0,0 +1,13 @@
+== Beta Release
+
+// tag::admonition[]
+[CAUTION]
+--
```

Comment:
The general convention is to use example block delimiters (`====`), but open block style seems to work fine.